.gitignore

```diff
@@ -24,3 +24,4 @@ npm-debug.log
 .mypy_cache/
 .vscode/
 cov.html
+**.DS_Store
```

Comment:
**note:** [In other Tribune repos](https://github.com/texastribune/texastribune/blob/master/.gitignore#L1-L5) we tend to do this individually at a local level using `.git/info/exclude` since it's not technically a file we'd all share. That said, it does still throw off new engineers (fellows especially)that are setting up their machines for the first time. TL;DR I don't mind having it here!

Reply:
Oh yeah good point... that's a goof on my part... I haven't had to mess with the .DS_Store since I first got here and I couldn't remember how I handled it then, but this jogged my memory on that. I wonder why they started showing up again for me... either way, not really relevant. I'll remove from here and handle it locally. Thanks for the reminder.

Reply:
I actually don't think I knew this 📝